styles/Events.module.scss

```diff
@@ -44,6 +44,15 @@
   grid-template-rows: 1fr 1fr 1rem;
   grid-template-columns: 200px 1fr;
   text-decoration: none;
+  transition: .3s ease;
+  &:hover{
+    color: $white;
+    background: $black;
+    box-shadow: inset 0 0 20px;
+    outline-color: rgba(49, 138, 172, 0);
+    outline-offset: 80px;
+    text-shadow: 1px 1px 6px $white;
```

Comment:
I think that text-shadow is not needed, @angelik0828 please confirm, here's some examples about what text-shadow is: https://developer.mozilla.org/en-US/docs/Web/CSS/text-shadow